src/govuk/components/footer/footer.yaml

```diff
@@ -67,6 +67,32 @@ params:
       type: object
       required: false
       description: HTML attributes (for example data attributes) to add to the anchor in the footer navigation section.
+- name: contentLicence
+  type: object
+  required: false
+  description: Object containing options for the content licence.
+  params:
+  - name: text
+    type: string
+    required: false
+    description: If `html` is set, this is not required. If `html` is provided, the `text` option will be ignored. If neither are provided, the text for the Open Government Licence is used.
+  - name: html
+    type: string
+    required: false
+    description: If `text` is set, this is not required. If `html` is provided, the `text` option will be ignored. If neither are provided, the text for the Open Government Licence is used.
```

Comment:
It looks to me like any HTML content provided here will be inserted within a `span` element, e.g:

```
<span class="govuk-footer__licence-description">[contentLicence.html here]</span>
```

Do we need to do something [similar to the accordion component yaml ](https://github.com/alphagov/govuk-frontend/blob/main/src/govuk/components/accordion/accordion.yaml#L30)and explain that only [phrasing content](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-span-element) should be passed in?

Reply:
Makes sense, I wasn't sure if that was a thing covered by the parameter docs. 

I'll add it for `copyright.html` too, as that's presented within a link that isn't configurable/removable in this version of the work.

Reply:
Have added text similar to what appears on the accordion.

The copyright instance was a little funky as there are various phrasing content elements that don't make sense within the link there (a, button, iframe, input (and other form elements), label). I've tried to make this a little clearer by making it "non-interactive phrasing content", but it might be better put as "you can only add text formatting elements".

Reply:
Maybe one for @claireashworth to have a quick look at?

Reply:
Are you referring to lines 78 and 82? If so they read clearly to me.

Reply:
If you are adding more to the description, the second option of only adding text formatting elements is better.

Reply:
I've updated the sentence to read "The copyright notice is inside an `<a>` element, so you can only use text formatting elements within it."